pages/docs/reference/map-operations.md

```diff
@@ -0,0 +1,285 @@
+
+```kotlin
+fun main() {
+//sampleStart
+    val numbersMap = mapOf("key1" to 1, "key2" to 2, "key3" to 3, "key11" to 11)
+    val filteredKeysMap = numbersMap.filterKeys { it.endsWith("1") }
+    val filteredValuesMap = numbersMap.filterValues { it < 10 }
+
+    println(filteredKeysMap)
+    println(filteredValuesMap)
+//sampleEnd
+}
+
+```
+</div>
+
+## `plus` and `minus` operators
+
+Due to the key access to elements, [`plus`](/api/latest/jvm/stdlib/kotlin.collections/plus.html) (`+`) and [`minus`](/api/latest/jvm/stdlib/kotlin.collections/minus.html) (`-`) operators work for maps differently than for other collections. 
+`plus` returns a `Map` that contains elements of its both operands: a `Map` on the left and a `Pair` or another `Map` on the right.
+When the right-hand side operand contains entries with keys present in the left-hand side `Map`, the result map contains the entries from the right side.
+
+<div class="sample" markdown="1" theme="idea" data-min-compiler-version="1.3">
+
+```kotlin
+fun main() {
+//sampleStart
+    val numbersMap = mapOf("one" to 1, "two" to 2, "three" to 3)
+    println(numbersMap + Pair("four", 4))
+    println(numbersMap + Pair("one", 10))
+    println(numbersMap + mapOf("five" to 5, "one" to 11))
+//sampleEnd
+}
+
+```
+</div>
+
+`minus` creates a `Map` from entries of a `Map` on the left except those with keys from the right-hand side operand.
+So, the right-hand side operand can be either a single key or a collection of keys: list, set, and so on.
+
+<div class="sample" markdown="1" theme="idea" data-min-compiler-version="1.3">
+
+```kotlin
+fun main() {
+//sampleStart
+    val numbersMap = mapOf("one" to 1, "two" to 2, "three" to 3)
+    println(numbersMap - "one")
+    println(numbersMap - listOf("two", "four"))
+//sampleEnd
+}
+
+```
+</div>
+
+For details on using [`plusAssign`](/api/latest/jvm/stdlib/kotlin.collections/plus-assign.html) (`+=`) and [`minusAssign`](/api/latest/jvm/stdlib/kotlin.collections/minus-assign.html) (`-=`) operators on mutable maps, see [Map write operations](#map-write-operations) below.
+
+## Map write operations
+
+[Mutable](collections-overview.html#collection-types) maps offer map-specific write operations.
+These operations let you change the map content using the key-based access to the values.
+
+There are certain rules that define write operations on maps:
+
+* Values can be updated. In turn, keys never change: once you add an entry, its key is constant.
+* For each key, there is always a single value associated with it. You can add and remove whole entries.
+
+Below are descriptions of the standard library functions for write operations available on mutable maps.
+
+### Adding and updating entries
+
+To  add a new key-value pair to a mutable map, use [`put()`](/api/latest/jvm/stdlib/kotlin.collections/-mutable-map/put.html).
+In `LinkedHashMap`, which is the default map implementation, new entries are added so that they are in the end when iterating the map.
```

Comment:
It's important to emphasize "new entries", because if `put` updates the existing entry, it remains in the same position.